justfile

```diff
@@ -149,31 +146,16 @@ build-images:
 	@just _build-bolt-boost
 
 # build the docker image for the bolt sidecar
+[private]
 _build-sidecar:
```

Comment:
According to the docs https://just.systems/man/en/private-recipes.html the attribute and the name with a leading underscore are equivalent. Can you choose one?